### DISCUSSION TITLE
Repositorio TP

### DISCUSSION THREAD

**Discussion by brianUtn98** [dds-jv/foro#124] [TPA - 2020] [answered]
¡Hola! Con mi grupo teníamos una pregunta: 
¿El TpA debemos subirlo al repositorio asignado, creando una carpeta aparte de la que ya está (PruebaDeConcepto)? ¿O debemos borrar esa carpeta y usar el repositorio solo para TPA?
Gracias.

**Comment by gastonprieto** [comment]
Buenas!

Si es en el mismo repositorio, pero no hace falta que dejen lo viejo, ya que hicieron un tag y se puede ver el historial, podemos corregir lo viejo sin problemas.

**Comment by gastonprieto** [comment]
te sirvio @brianUtn98 ?

**Comment by brianUtn98** [comment, marked as answer]
Sí @gastonprieto! Muchas gracias y perdón por colgar la respuesta.
Saludos

**Comment by brianUtn98** [comment]
Wow, despertaron recuerdos de vietnam
